<ui version="4.0" >
 <class>ErrorDialog</class>
 <widget class="QDialog" name="ErrorDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Error</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Application Error !</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>467</width>
        <height>554</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>&amp;Description</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="uiErrorTitle" >
         <property name="text" >
          <string>Application Error</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string/>
           </property>
           <property name="pixmap" >
            <pixmap resource="common.qrc" >:/images/warning.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" >
           <item>
            <widget class="QLabel" name="label_3" >
             <property name="text" >
              <string>&lt;b>Error code:&lt;/b></string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="uiErrorInfo" >
             <property name="text" >
              <string>View details</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>467</width>
        <height>554</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>De&amp;tails</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QTextBrowser" name="uiDetails" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushAccept" >
       <property name="text" >
        <string>&amp;Accept</string>
       </property>
       <property name="icon" >
        <iconset resource="common.qrc" >:/images/ok.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="common.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>pushAccept</sender>
   <signal>clicked()</signal>
   <receiver>ErrorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>410</x>
     <y>645</y>
    </hint>
    <hint type="destinationlabel" >
     <x>392</x>
     <y>665</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
